### PR TITLE
[specific ci=6-07-Create-Network] Add bridge to public network ping test

### PR DESCRIPTION
**Pull request by rajanashok**
Adding test to verify if ping works from a bridge to a public network.

### CODE REVIEW

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot [hunk 1]
```diff
@@ -96,6 +96,70 @@ Management network - valid
...
+
+    Log To Console  Create a bridge portgroup
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 bridge
+    Should Be Empty  ${out}
+
+    Log To Console  Create a management portgroup.
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 management
+    Should Be Empty  ${out}
+
+    ${output}=  Run  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} --force=true --bridge-network=bridge --public-network=vm-network --compute-resource=%{TEST_RESOURCE} --container-network management --container-network vm-network --container-network-ip-range=management:10.10.10.0/24 --container-network-gateway=management:10.10.10.1/24 --no-tlsverify
```
Comment (hmahmood) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: Do we need the management network for this test?

Comment (rajanashok) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: Not needed for this test, but testing the superset functionality on this so I can plug this in for the bridge to container ping test.

Comment (rajanashok) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: @hmahmood Added both bridge to public and bridge to management test, and it passed the CI. Please review when you get a chance. Thanks.

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot [hunk 2]
```diff
@@ -96,6 +96,112 @@ Management network - valid
...
+    Should Not Contain  ${output}  Error:
+
+    Log To Console  Getting IP for management container
+    ${ip}=  Run  docker %{VCH-PARAMS} inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress }}{{end}}' m1
+
+    Log To Console  Pinging from bridge to management container.
+    ${id}=  Run  docker %{VCH-PARAMS} run -d busybox ping -c 30 ${ip}
+
+    Log To Console  Attach to running container.
+    ${out}=  Run  docker %{VCH-PARAMS} attach ${id}
```
Comment (hmahmood) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: I think we have used `docker logs --follow` previously here to get the output of the container process.

**Pull request merged**